src/sentry/api/endpoints/sentry_app_avatar.py

```diff
@@ -1,17 +1,13 @@
 from sentry.api.bases import SentryAppBaseEndpoint
 from sentry.api.bases.avatar import AvatarMixin
+from sentry.api.serializers.rest_framework.sentry_app import SentryAppAvatarSerializer
 from sentry.models import SentryAppAvatar
 
 
 class SentryAppAvatarEndpoint(AvatarMixin, SentryAppBaseEndpoint):
     object_type = "sentry_app"
     model = SentryAppAvatar
-
-    def get(self, request, **kwargs):
-        return super().get(request, access=request.access, **kwargs)
-
-    def put(self, request, **kwargs):
-        return super().put(request, access=request.access, **kwargs)
```

Comment:
These methods previously contained feature flag checks, now they can be safely removed since they just call `super()` anyway.

Reply:
i was wrong oops